lib/user_comm_screener.rb

```diff
@@ -99,7 +99,7 @@ def disallowing_pms?(user_id)
   def initialize(acting_user: nil, acting_user_id: nil, target_user_ids:)
     raise ArgumentError if acting_user.blank? && acting_user_id.blank?
     @acting_user = acting_user.present? ? acting_user : User.find(acting_user_id)
-    @target_users = User.where(id: target_user_ids).pluck(:id, :username).to_h
+    @target_users = User.where(id: target_user_ids).where.not(id: @acting_user.id).pluck(:id, :username).to_h
```

Comment:
Just a thought I have but I did wonder if it'll be more efficient to just exclude the id from the array instead of adding another filter to the query.

Reply:
Yeah was wondering that, will do